### ISSUE TITLE
Event hub triggered functions do not work with bundles on V4

### ISSUE THREAD

**Issue by paulbatum** [Azure/azure-functions-host#8119]
Python, JavaScript, PowerShell and Java function apps on V4 will encounter startup errors when using event hub triggered functions.

This is because the V4 functions host requires eventhub extension 4.3.0 or later (this check was added [here](https://github.com/Azure/azure-functions-host/pull/7816), because of https://github.com/Azure/Azure-Functions/issues/2072). 

However, it appears that neither the 2.x bundle nor the 3.x bundle contain this version of the extension. The latest deployed 2.x bundle is [2.8.3](https://github.com/Azure/azure-functions-extension-bundles/releases/tag/2.8.3), which has 4.2.0. The latest deployed 3.x bundle is [3.0.0](https://github.com/Azure/azure-functions-extension-bundles/releases/tag/3.0.0), which has 4.1.1. This is based on browsing the bundles dir in Central US.

#### Investigative information
```
// central us
FunctionsLogs
| where TIMESTAMP >= datetime(2022-02-04T23:48:50.7021611Z) and TIMESTAMP <= datetime(2022-02-04T23:54:53.6353377Z)
| where AppName == "pbatum-v4-js"
| where Level < 4
```
#### Repro steps

1. Create a JavaScript function app through the portal. (by default it will be a V4 app configured to use the 2.x bundle)
2. Add an event hub triggered function. 
3. Try to hit the root of the app, or refresh the main page in the portal. You should see the following error:

![image](https://user-images.githubusercontent.com/10663/152620811-b3597e96-9ad8-491e-8bbf-594580f49acb.png)

#### Known workarounds

Force the host to use a later bundle version that has not been deployed globally yet, e.g. use the following host.json to use the 2.8.4 bundle.

```json
{
  "version": "2.0",
  "extensionBundle": {
    "id": "Microsoft.Azure.Functions.ExtensionBundle",
    "version": "[2.8.4, 3.0.0)"
  }
}
```

For the 3.x bundle, 3.3.0 works:

```json
{
  "version": "2.0",
  "extensionBundle": {
    "id": "Microsoft.Azure.Functions.ExtensionBundle",
    "version": "[3.3.0, 4.0.0)"
  }
}
```

**Comment by jimbobbennett** [comment]
Is this issue going to be fixed? Today I created a new event hub trigger function app and the host.json had the version set to:

```json
"extensionBundle": {
  "id": "Microsoft.Azure.Functions.ExtensionBundle",
  "version": "[2.*, 3.0.0)"
}
```

Giving the above error in the portal and the function app not running

**Comment by paulbatum** [comment]
@jimbobbennett My application recieved the fix a few days ago. I was able to put the host.json back to the default (what you shared above) and it still works.

I was able to confirm that it was fixed by a bundle release by going to the debug console in kudu and checking the bundles present on the machine. 2.9.1 is there now.

![image](https://user-images.githubusercontent.com/10663/159537590-3e8c6d9b-96ae-4469-be38-922d58a2f7f7.png)

If the same is not true for you, my guess is that global rollout is still in progress (my app is in Central US).
